build/dev-runtime.js

```diff
@@ -86,6 +86,8 @@ module.exports.DevelopmentRuntime = function(
   this.run = async function reloadProc() {
     const childPort = await getPort();
     const command = `
+      process.on('SIGTERM', () => process.exit());
```

Comment:
Why is this needed now, but not before? Just looking for some additional context.

Reply:
Without this, a zombie process was left each time dev was run.